hack/doc/README.md

```diff
@@ -0,0 +1,26 @@
+# role-permissions2md - YAML to Markdown converter for operator permissions
+
+This script can be used to create a table containing permissions needed by components of the Dynatrace Kubernetes Operator. The resulting tables can be used as input for SUS tickets to update our public documentation here: https://www.dynatrace.com/support/help/setup-and-configuration/setup-on-container-platforms/kubernetes/get-started-with-kubernetes-monitoring/dt-component-permissions#dto
```

Comment:
nitpick: `SUS tickets` is internal specific thing I don't think we need to mention that in a public readme, if we want to mention this we can just say `The resulting tables can be used as input to update our public documentation here: ...`